fine_tuning_sql/notebooks/1_download_data.ipynb

```diff
@@ -5,26 +5,73 @@
    "metadata": {},
    "source": [
     "# Part 1: Download data\n",
-    "This notebook goes over data download for SQL model fine-tuning"
+    "This notebook goes over data download for SQL model [Pretraining](#pretraining-dataset) and [Fine-tuning](#fine-tunning-dataset)\n",
+    "\n",
+    "We recommend you going directly to the [Fine-tuning](#fine-tunning-dataset) if you want to use as base model an instruct model like Llama-7b-Instruct-hf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pretraining dataset\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Pretraining dataset: Load and save \n"
+    "### Small stack smol dataset for SQL Pretraining\n",
```

Comment:
Small stack is just for testing with small subset. The realy recipe should use the Full squad dataset for SQL.

Reply:
Explanation added